tests/test_intersection_over_union.py

```diff
@@ -4,23 +4,36 @@
 
 def test_iou():
     
-    size = 10
-    a = np.arange(1, size, 1, dtype=np.uint8)
-    b = np.arange(1, size, 1, dtype=np.uint8)
+    # Create two arrays of random length
+    size = np.random.randint(5, 10)
+    low = 1
+    a = np.arange(low, size, 1, dtype=np.uint8)
+    b = np.arange(low, size, 1, dtype=np.uint8)
+    
+    # Shuffle array
     np.random.shuffle(b)
     
+    # delete random entries from a and b
+    idx = np.random.randint(low, size-1, 1)
+    b[idx] = 0
+    a[idx] = 0
+    
     iou = biau.label.intersection_over_union_matrix(a, b)
     
-    assert(iou.shape == (size, size))
+    assert(iou.shape == (len(np.unique([a, b])), len(np.unique([a, b]))))
 
 def test_intersection_over_union_matrix():
     a = np.asarray([1, 1, 2, 2, 0, 0])
     b = np.asarray([3, 2, 2, 0, 0, 1])
 
     reference = np.asarray([[0.33, 0.5,  0.,   0.  ],
                             [0.,   0.,   0.33, 0.5 ],
-                            [0.33, 0.,   0.33, 0.  ]])
+                            [0.33, 0.,   0.33, 0.  ],
+                            [0.,   0.,   0.,   0.  ]])
 
     iou = biau.intersection_over_union_matrix(a, b)
 
     assert np.allclose(iou, reference, 0.02)
+    
+if __name__ == "__main__":
+    test_iou()
```

Comment:
Please add a line break by the end of every file.
![image](https://user-images.githubusercontent.com/12660498/141107474-25eeb221-71e3-4888-a0d1-18bc9f8041c6.png)

Reply:
Changed in 3c94fb2090ab463cd761e67c331a8298a0169127